communication.rst

```diff
@@ -169,12 +169,19 @@ from that particular thread.
 
 Customising notifications on user preference
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+<<<<<<< HEAD
```

Comment:
Not sure what happened here, but it looks like a bad merge

Reply:
Already fixed in 35de0e7bb87f9f66f723b2d69e4258daf13b768a.